File: code-coaches/easy/duct-tape/duct_tape.rb
def get_area(height, width)
  area = height * width
end

def to_inches(feet)
  inches = feet * 12
end

def get_duct_tapes_needed(height, width)
  # Everything is calculated in inches
  duct_tape_height = 720
  duct_tape_width = 2

  door_area_both_sides = get_area(height, width) * 2
  duct_tape_area = get_area(duct_tape_height, duct_tape_width)

  duct_tapes_needed = (door_area_both_sides.to_f / duct_tape_area).ceil
end

def main
  door_height = gets.chomp.to_i
  door_width = gets.chomp.to_i
  puts get_duct_tapes_needed(to_inches(door_height), to_inches(door_width))
end

if __FILE__ == $0
  main
end

File: code-coaches/easy/duct-tape/duct_tape.c
#include <stdio.h>
#include <math.h>

int get_area(int height, int width)
{
    return height * width;
}

int to_inches(int feet)
{
    return feet * 12;
}

int get_duct_tapes_needed(int height, int width)
{
    // Everything is calculated in inches
    int duct_tape_height = 720;
    int duct_tape_width = 2;

    int door_area_both_sides = get_area(height, width) * 2;
    int duct_tape_area = get_area(duct_tape_height, duct_tape_width);

    int duct_tapes_needed = ceil((double) door_area_both_sides / duct_tape_area);
    return duct_tapes_needed;
}

int main()
{
    int door_height_feet, door_width_feet;

    scanf("%d%d", &door_height_feet, &door_width_feet);

    int door_height_inches = to_inches(door_height_feet);
    int door_width_inches = to_inches(door_width_feet);

    printf("%d", get_duct_tapes_needed(door_height_inches, door_width_inches));

    return 0;
}
